frontend/src/utils/beep.tsx

```diff
@@ -0,0 +1,8 @@
+const beep = () => {
+  const snd = new Audio(
```

Comment:
I wonder if there's a better way to include this, rather than write the audio out explicitly as a base64?

Reply:
@amanape Which file format would be better?

Reply:
@SmartManoj There are libraries that you can install that provide sound files, like icons. Though without a library, I would probably move this into its own file and import it